src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -375,6 +375,11 @@ ThumbnailHeight =
 # Default: false
 ChunkedTransfer = 
 
+# Overrides the FFmpeg Audio options in UMS for this renderer.
+# An empty value means that the default UMS settings will be used.
+# Default: ""
+CustomAudioFFmpegOptions =
```

Comment:
It would be good to rename this to `CustomFFmpegAudioOptions` to match our other conventions